python/pyspark/sql/functions.py

```diff
@@ -200,17 +226,20 @@ def _():
 @since(1.3)
 def approxCountDistinct(col, rsd=None):
     """
-    .. note:: Deprecated in 2.1, use approx_count_distinct instead.
+    .. note:: Deprecated in 2.1, use :func:`approx_count_distinct` instead.
     """
     return approx_count_distinct(col, rsd)
 
 
 @since(2.1)
-def approx_count_distinct(col, rsd=None):
-    """Returns a new :class:`Column` for approximate distinct count of ``col``.
+def approx_count_distinct(col, rsd=0.05):
```

Comment:
Okay, I double checked some APIs have them but let's leave this out. It sounds requiring fixing/checking logic below and the deprecated one above. I assume this should be doc only change PR.